webpack/components/QuestionWidget.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+var routes = require("../routes.js");
+if(routes.Routes){ routes=routes.Routes;}
```

Comment:
Not a huge fan of replacing the routes variable.
How is this supposed to work if routes.Routes isn't there?
My thought is to pull the route importing into question_list.js. Then pass it down as a prop on QuestionList and then to QuestionWidget. You can then declare a PropType on both components and declare that the question_path function needs to be there.